include/swift/SIL/SILInstruction.h

```diff
@@ -1806,6 +1806,50 @@ class TailAllocatedDebugVariable {
 static_assert(sizeof(TailAllocatedDebugVariable) == 4,
               "SILNode inline bitfield needs updating");
 
+class SILDebugVariableSupplement {
+protected:
+  struct SourceLocKind {
+    static constexpr unsigned Loc = 0b01;
+    static constexpr unsigned Scope = 0b10;
+  };
+
+  unsigned NumDIExprOperands;
```

Comment:
We probably don't need to allow much more than 2^8 DIExpr operands, in case we are short on space.